src/pluto/pluto.py

```diff
@@ -395,17 +396,56 @@ def create_null(df_null, v1, v2, v3, condo, mapped):
                 "pfirm15_flag",
             ]
 
-            def generate_graph(r, total, title):
-                y = [r[i] for i in x]
-                text = [f"{round(r[i]/total*100, 2)}%" for i in x]
-                return go.Scatter(
-                    x=x,
-                    y=y,
-                    mode="lines",
-                    name=title,
-                    hovertemplate="<b>%{x} %{text}</b>",
-                    text=text,
+            field_range = st.slider(
+                "Select a range of values",
+                min_value=0,
+                max_value=len(x),
+                value=[0, 12],
+            )
+
+            def sort_and_filter_df(df, range):
+                df = df[["pair"] + df.columns.tolist()[range[0] + 1 : range[1] + 1]]
+                df_transformed = (
+                    df.set_index("pair")
+                    .stack()
+                    .reset_index()
+                    .rename(
+                        columns={"pair": "version", "level_1": "field", 0: "change"}
+                    )
+                    .sort_values(by="change", ascending=False)
                 )
+                return df_transformed
+
+            def display_graph(df_transformed, range, grouped=True):
```

Comment:
I think it might be nice to turn this horizontal, it's a little jarring for the user to see a different type of plot between the mismatch and aggregate graphs. So we could move this above/below to keep the similarly formatted plots, or turn it horizontal, I think both would be less jarring